packages/devtools_server/lib/src/handlers.dart

```diff
@@ -103,3 +177,182 @@ class ServerApi {
   FutureOr<shelf.Response> notImplemented(shelf.Request request) =>
       shelf.Response(204);
 }
+
+// Access the ~/.flutter file.
+class Usage {
```

Comment:
nit: rename to FlutterUsage to indicate this is the Flutter file.

Reply:
Done.